reyna/src/com/b2msolutions/reyna/blackout/Time.java

```diff
@@ -39,4 +39,8 @@ public boolean isBeforeOrEqualTo(Time other) {
     public boolean isAfterOrEqualTo(Time other) {
         return this.minuteOfDay >= other.minuteOfDay;
     }
+
+    public long getCurrentTimeMillis() {
+        return System.currentTimeMillis();
```

Comment:
As I understand it the concern here is that the tie is in the blackout package and is not specifically for blacking out in this  instance. In that case you COULD argue that it should never have had a time class in the blackout package and that it should always have been more generalised if it could be used elsewhere. Personally although it is not perfect design I don't really have much of a problem with calling the existing time class, even if it is in a package named blackout.

It certainly cannot be static in this instance due to mocking as already mentioned. Also having a time class with just one method in it seems a little excessive.

I would probably leave it as is.